packages/TelegramClient-Core.package/TCCMessage.class/instance/userId..st

```diff
@@ -0,0 +1,4 @@
+accessing
+userId: aNumber
```

Comment:
why rename to `userId`?

Reply:
because we will never store something else as a userId in it

Reply:
I am just a bit confused, what could `senderId` be, other than a user id? What did we store in that so far?